### PR TITLE
[SW-1526] PySparkling cannot parse embedded version.txt file

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

**Comment by mn-mikke** [comment]
@jakubhava Mering to master in order to create a new nightly build. If you find any drawbacks, I will fix them in a follow up PR.

File: py/ai/h2o/sparkling/__init__.py [hunk 1]
```diff
@@ -18,3 +18,4 @@
 from ai.h2o.sparkling.H2OConf import H2OConf
 from ai.h2o.sparkling.H2OContext import H2OContext
 from ai.h2o.sparkling.Initializer import Initializer
+from ai.h2o.sparkling.VersionComponents import VersionComponents
```
Comment (jakubhava) [inline py/ai/h2o/sparkling/__init__.py]: nl

File: py/ai/h2o/sparkling/VersionComponents.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import re
+
+class VersionComponents(object):
+
+    @staticmethod
+    def parseFromVersion(version):
```
Comment (jakubhava) [inline py/ai/h2o/sparkling/VersionComponents.py]: Would be good to have tests for this method to verify the the regexp for normal & nightly version

Comment (mn-mikke) [reply py/ai/h2o/sparkling/VersionComponents.py]: I will add them, thanks!

**Pull request merged**